packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -789,20 +789,16 @@ export default {
   border-radius: var(--dt-size-radius-400);
   border: var(--dt-size-border-100) solid;
   border-color: var(--dt-color-border-default);
-  box-shadow: 0 0 0 0 rgba(0, 0, 0, 0%);
   line-height: var(--dt-font-line-height-400);
   cursor: text;
-  transition: all 100ms cubic-bezier(0.22, 1, 0.36, 1);
+  opacity: 1;
```

Comment:
```suggestion
  opacity: var(--dt-opacity-1300);
```

Reply:
Thanks, ultimately removed this altogether because it wasn't needed anymore.